packages/protocol/contracts/L1/TaikoEvents.sol

```diff
@@ -30,13 +30,13 @@ abstract contract TaikoEvents {
     /// @param prover The prover whose transition is used for verifying the
     /// block.
     /// @param blockHash The hash of the verified block.
-    /// @param stateRoot The block's state root.
+    /// @param transitionId The transition that used to verify this block.
     /// @param tier The tier ID of the proof.
     event BlockVerified(
         uint256 indexed blockId,
         address indexed prover,
         bytes32 blockHash,
-        bytes32 stateRoot,
+        uint32 transitionId,
```

Comment:
I think we should keep the type here, or all clients will break. This will be an ABI break change.